server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -16,6 +16,7 @@
 // under the License.
 package com.cloud.vm;
 
+import static com.cloud.configuration.ConfigurationManagerImpl.VM_USERDATA_MAX_LENGTH;
```

Comment:
@shwstppr generally bad practice to import/depend on an `impl`; instead refactor the static field from an interface/service/manager than an impl class. If this value is determined/depends on a configkey, maybe use a getter. Otherwise LGTM.